libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccountBuilder.java

```diff
@@ -338,6 +340,17 @@ public UserAccountBuilder csrfToken(String csrfToken) {
         return this;
     }
 
+    /**
+     * Sets if the user authenticated with native login.
+     *
+     * @param nativeLogin if the user authenticated with native login.
+     * @return Instance of this class.
+     */
+    public UserAccountBuilder nativeLogin(Boolean nativeLogin) {
```

Comment:
Wouldn't it be better to call it setNativeLogin??

Reply:
I don't have a problem renaming it, but since it's a builder class all of the setters are named this way.

Reply:
That's fine keeping it the same. May have the param name change to isNativeLogin??